components/emailSignature/emailSignature.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react'
+import { SignatureData } from '../../lib/types'
+import { icons } from '../../styles/icons'
+import { theme } from '../../styles/theme'
+import { ThemeProvider } from '@mui/material'
+
+interface EmailSignatureProps {
+  signatureData?: SignatureData
+}
+
+// This component now uses inline styles so that it can be sent over Mailgun.
+// See styles definition below
+export default function EmailSignature ({ signatureData }: EmailSignatureProps) {
+  return <>
+    {signatureData &&
+      (
+        <ThemeProvider theme={theme}>
+          <table cellPadding={0} cellSpacing={0}
+            style={styles.table}>
+            <tbody>
+              <tr>
+                <td valign="top" style={styles.logoContainer}>
+                  <img
+                    id="preview-image-url"
+                    src={icons.HBP_LOGO.image}
+                    alt={icons.HBP_LOGO.altText}
+                    style={styles.logoImage}
+                  />
+                </td>
+                <td style={styles.contentContainer}>
+                  <table cellPadding={0} cellSpacing={0}>
+                    <tbody>
+                      <tr>
+                        <td colSpan={2} style={styles.signatureName}>
+                          {signatureData.fullName}
+                        </td>
+                      </tr>
+                      <tr>
+                        <td colSpan={2} style={styles.signatureText}>
+                          {signatureData.title}
+                        </td>
+                      </tr>
+                      <tr>
+                        <td colSpan={2} style={styles.signatureText}>
+                          <strong>HackBeanpot, Inc.</strong>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td style={styles.phoneNumber}>
+                          {signatureData.phone}
```

Comment:
Nit but can we add a TODO to style phone numbers XXX-XXX-XXXX? Noticed this is not a thing in your screenshot